packages/@aws-cdk/pipelines/README.md

```diff
@@ -38,10 +55,38 @@ will work, see the section **CDK Environment Bootstrapping** below).
   * to this table in its properties.
   */
 import { DatabaseStack, ComputeStack } from '../lib/my-stacks';
-
 import { Construct, Stage, Stack, StackProps, StageProps } from '@aws-cdk/core';
-import { CdkPipeline } from '@aws-cdk/pipelines';
-import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import { CodePipeline, CodePipelineSource, ScriptStep } from '@aws-cdk/pipelines';
+
+/**
+ * Stack to hold the pipeline
+ */
+class MyPipelineStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const pipeline = new CodePipeline(this, 'Pipeline', {
+      synth: new ScriptStep('Synth', {
+        // Will use a SecretsManager secret named 'github-token' by default
+        input: CodePipelineSource.gitHub('my-org/my-app'),
+        commands: [
+          'npm ci',
+          'npm run build',
+          'npx cdk synth',
+        ],
+      }),
+    });
+
+    // Do this as many times as necessary with any account and region
+    // Account and region may different from the pipeline's.
+    pipeline.addStage(new MyApplication(this, 'Prod', {
```

Comment:
Mention where `MyApplication` is coming from or event add some initial code that shows how it's defined.